lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -264,21 +262,24 @@ defmodule ExUnit.Runner do
             end
           end)
 
-        send(parent, {self(), :test_finished, %{test | time: us}})
+        send(parent_pid, {self(), :test_finished, %{test | time: us}})
         exit(:shutdown)
       end)
 
-    timeout = get_timeout(test.tags, config)
-    test = receive_test_reply(test, test_pid, test_ref, timeout)
+    reply_config = %{
+      pid: test_pid,
+      ref: test_ref,
+      timeout: timeout
+    }
```

Comment:
Can we please keep the previous call? It is fine for us to call a function with four arguments. I don't think it makes much sense to build a map just to unpack it right after.